google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/FakeBigQueryWrite.java

```diff
@@ -91,6 +91,10 @@ public void setTimesToClose(long numberTimesToClose) {
     serviceImpl.setTimesToClose(numberTimesToClose);
   }
 
+  public void setCloseForeverAfter(long closeForeverAfter) {
```

Comment:
I get this is a fake, but I'm not clear from the name what it's purpose is.  A better name might make sense here or a comment clarifying usage.